app/controllers/api/v1/locations_controller.rb

```diff
@@ -19,11 +19,10 @@ def index
 
       def show
         location = Location.includes(
-          contacts: :phones,
-          services: %i[categories contacts phones regular_schedules
-                       holiday_schedules]
-        ).find(params[:id])
-        render json: location, status: :ok if stale?(location, public: true)
+          contacts: :phones
+          ).find(params[:id])
+
```

Comment:
Why did we remove this line including location services in the query? This might be what's breaking the spec?

Reply:
Eric and I moved into the locations serializer in order to utilize the scope method added to the services model to deal with this archiving services feature.

Reply:
ah ok!